metrics/internal/cache/pv.go

```diff
@@ -0,0 +1,152 @@
+package cache
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"sync"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/watch"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/klog"
+)
+
+// Cache metadata for all PV with annotation - pv.kubernetes.io/provisioned-by: rook-ceph.rbd.csi.ceph.com
```

Comment:
it should be `openshift-storage.rbd.csi.ceph.com`

Reply:
I have updated the check to verify if suffix is `.rbd.csi.ceph.com`. Since, first part might change.
Will update comment accordingly.